From 14e5a2b03a580d4f706c4ab2b2f0d9a3c825a1cb Mon Sep 17 00:00:00 2001
From: Scott Hess <shess@chromium.org>
Date: Tue, 14 Mar 2017 13:45:46 -0700
Subject: [PATCH 09/10] [build] Fix _CRT_RAND_S conflict in sqliteInt.h

Chromium already defines this, for the same reasons that SQLite does,
but it conflicts with SQLite's definition.

BUG=701522
---
 third_party/sqlite/src/src/sqliteInt.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/third_party/sqlite/src/src/sqliteInt.h b/third_party/sqlite/src/src/sqliteInt.h
index 05b691b32dee..fcbcf8bb6a40 100644
--- a/third_party/sqlite/src/src/sqliteInt.h
+++ b/third_party/sqlite/src/src/sqliteInt.h
@@ -55,8 +55,11 @@
 ** or higher.
 */
 #if defined(_MSC_VER) && _MSC_VER>=1400
+/* TODO(shess): Already defined by build/config/win/BUILD.gn */
+#ifndef _CRT_RAND_S
 #  define _CRT_RAND_S
 #endif
+#endif
 
 /*
 ** Include the header file used to customize the compiler options for MSVC.
-- 
2.13.1.518.g3df882009-goog

